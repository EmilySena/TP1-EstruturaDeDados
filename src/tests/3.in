3
5874
3256
8412
0
-2
0
3256
8412
-3
-1
-2
0