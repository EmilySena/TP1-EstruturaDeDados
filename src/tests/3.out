nave 8412 em combate
3256
5874
nave 3256 em combate
nave 3256 avariada
nave 8412 avariada
3256
8412
nave 3256 consertada
3256
5874
nave 3256 em combate
